### ISSUE TITLE
Error building console / missing image?

### ISSUE THREAD

**Issue by dazld** [pointslope/docker-datomic-example#6]
```
$ docker-compose up
Building console
Step 1 : FROM pointslope/docker-datomic-console:0.9.5385-alpine
Pulling repository docker.io/pointslope/docker-datomic-console
ERROR: Service 'console' failed to build: Error: image pointslope/docker-datomic-console:0.9.5385-alpine not found
```

Any ideas..? Everything else looks like it built ok.

**Comment by christianromney** [comment]
I'll double check the tag on docker hub. stay tuned.

**Comment by dazld** [comment]
ah! found it - typo in the docker image name. should be:

```
FROM pointslope/datomic-console:0.9.5385-alpine
# the hostname 'db' is created by linking a Datomic Pro Starter container with the db alias
# docker-compose does this for us, but it can be done with docker run --link
CMD ["dev", "datomic:dev://datomicdb:4334"]
```

not `docker-datomic-console`

**Comment by christianromney** [comment]
i thought i pushed that change sorry

**Comment by dazld** [comment]
no probs, thanks for quick reply.

**Comment by christianromney** [comment]
My pleasure. Thanks for reporting this, and best of luck!

**Comment by christianromney** [comment]
Fixed as of 9f586fb
